skaffold.yaml

```diff
@@ -50,4 +50,4 @@ deploy:
         packaged: null
         imageStrategy:
           fqn: null
-          helm: null
```

Comment:
Probably this file was accidentally changed as part of this PR?

Reply:
Done